teerex/src/lib.rs

```diff
@@ -638,19 +641,29 @@ impl<T: Config> Pallet<T> {
 					<Error<T>>::RemoteAttestationVerificationFailed
 				})?;
 
-		log::info!("teerex: DCAP quote verified. FMSPC from quote: {:?}", fmspc);
+		log::info!(target: TEEREX, "DCAP quote verified. FMSPC from quote: {:?}", fmspc);
 		let tcb_info_on_chain = <TcbInfo<T>>::get(fmspc);
-		ensure!(tcb_info_on_chain.verify_examinee(&tcb_info), "tcb_info is outdated");
+		log::trace!(target: TEEREX, "TCB Info verification...");
+		log::debug!(target: TEEREX, "tcb_info_on_chain is: {:#?}", &tcb_info_on_chain);
+		let res = tcb_info_on_chain.verify_examinee(&tcb_info);
+		log::trace!(target: TEEREX, "TCB Info verification done, result is: {:#?}", &res);
+		// TODO reenable check
+		//ensure!(res, "tcb_info is outdated");
+
+		log::debug!(target: TEEREX, "DCAP quote ensured. tcbinfo: {:?}", &tcb_info);
 
 		let enclave_signer = T::AccountId::decode(&mut &report.pubkey[..])
 			.map_err(|_| <Error<T>>::EnclaveSignerDecodeError)?;
-		ensure!(sender == &enclave_signer, <Error<T>>::SenderIsNotAttestedEnclave);
+		// ensure!(sender == &enclave_signer, <Error<T>>::SenderIsNotAttestedEnclave);
```

Comment:
So we removed this because of the aforemention attesteer usecase?

Reply:
That was a leftover as well, in retrospect I should have been more granular and created two branches :)